src/main/java/org/sagebionetworks/web/client/widget/evaluation/EvaluationList.java

```diff
@@ -32,13 +37,12 @@ public Widget asWidget() {
     return view.asWidget();
   }
 
-  public Evaluation getSelectedEvaluation() {
-    if (evaluationList.size() == 1) {
-      return evaluationList.get(0);
-    } // else
-    Integer selectedEvaluationIndex = view.getSelectedEvaluationIndex();
-    if (selectedEvaluationIndex == null) return null;
```

Comment:
The SRC component handles this logic now and returns the selected `Evaluation` rather than its index.